{"name": "测试用例4", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == None\n  +3\n  -None", "trace": "self = <test_calculator_Sunday.TestCalc object at 0x0000021D51EEDCA0>, a = 3, b = 0, expect = 3\ninit_calculator = <pythoncode.calculator.Calculator object at 0x0000021D51ECB820>\n\n    @allure.story('除法测试')\n    @allure.title('测试用例4')\n    @pytest.mark.run(order=4)\n    @pytest.mark.parametrize('a,b,expect',\n                             get_data()[0],\n                             ids=get_data()[1])\n    def test_div(self, a, b, expect, init_calculator):\n>       assert expect == init_calculator.div(a, b)\nE       assert 3 == None\nE         +3\nE         -None\n\ntest_calculator_Sunday.py:33: AssertionError"}, "parameters": [{"name": "a", "value": "3"}, {"name": "b", "value": "0"}, {"name": "expect", "value": "3"}], "start": 1607956740386, "stop": 1607956740387, "uuid": "f1e23d4b-68ef-4972-af89-d04c39671bf7", "historyId": "2ba3aeb484cbe668b70373ce802f6f10", "testCaseId": "9f5ed0824c945532a44497218933a2ab", "fullName": "test_calculator_Sunday.TestCalc#test_div", "labels": [{"name": "story", "value": "除法测试"}, {"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_calculator_Sunday"}, {"name": "subSuite", "value": "TestCalc"}, {"name": "host", "value": "wwxin1"}, {"name": "thread", "value": "56164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_calculator_Sunday"}]}